cypher/cypher-docs/src/docs/dev/execution-plans.asciidoc

```diff
@@ -52,89 +77,91 @@ This table comprises all the execution plan operators ordered lexicographically.
-| <<query-plan-roll-up-apply, RollUpApply>>      | Performs a nested loop. Executes a pattern expression or pattern comprehension. | |
-| <<query-plan-select-or-anti-semi-apply, SelectOrAntiSemiApply>>            | Performs a nested loop. Tests for the absence of a pattern predicate if an expression predicate evaluates to `false`. | |
-| <<query-plan-select-or-semi-apply, SelectOrSemiApply>>                     | Performs a nested loop. Tests for the presence of a pattern predicate if an expression predicate evaluates to `false`. | |
-| <<query-plan-semi-apply, SemiApply>>                                       | Performs a nested loop. Tests for the presence of a pattern predicate. | |
-| <<query-plan-set-labels, SetLabels>>                            | Sets labels on a node. | | Y
-| <<query-plan-set-node-property-from-map, SetNodePropertyFromMap>>          | Sets properties from a map on a node. | | Y
-| <<query-plan-set-property, SetProperty>>          | Sets a property on a node or relationship. | | Y
-| <<query-plan-set-relationship-property-from-map, SetRelationshipPropertyFromMap>>    | Sets properties from a map on a relationship. | | Y
-| <<query-plan-skip, Skip>>                                                  | Skips 'n' rows from the incoming rows. | |
-| <<query-plan-sort, Sort>>                                                  | Sorts rows by a provided key. | |
-| <<query-plan-top, Top>>                                                    | Returns the first 'n' rows sorted by a provided key. | |
-| <<query-plan-triadic-selection, TriadicSelection>>                                            | Solves triangular queries, such as the very common 'find my friend-of-friends that are not already my friend'. | |
-| <<query-plan-undirected-relationship-by-id-seek, UndirectedRelationshipByIdSeek>> | Reads one or more relationships by id from the relationship store. | Y |
-| <<query-plan-union, Union>>                                                | Concatenates the results from the right-hand operator with the results from the left-hand operator. | |
-| <<query-plan-unwind, Unwind>>                                              | Returns one row per item in a list. | |
-| <<query-plan-value-hash-join, ValueHashJoin>>                       | Executes a hash join on arbitrary values. | |
-| <<query-plan-varlength-expand-all, VarLengthExpand(All)>>                                   | Traverses variable-length relationships from a given node. | |
-| <<query-plan-varlength-expand-into, VarLengthExpand(Into)>>                                   | Finds all variable-length relationships between two nodes. | |
-| <<query-plan-varlength-expand-pruning, VarLengthExpand(Pruning)>>                                   | Traverses variable-length relationships from a given node and only returns unique end nodes. | |
+|Name                                                                       | Description | Leaf? | Updating? | Eager?
```

Comment:
I now have a better idea. Rename the column `Considerations`, and put the word `Eager` in each place you currently have a `Y`. This has two advantages:

* People reading the table way down, don't have to scroll up to see what the `Y` actually means
* We can deal with more subtle situations like partially eager operations (mostly a concern once we implement index-backed order-by, which is 3.5, but it is not wrong to have this forward compatible)